### DISCUSSION TITLE
Is there an overview of which Tree-sitter parsers are used by Github.com for syntax highlighting?

### DISCUSSION THREAD

**Discussion by Hubro** [tree-sitter/tree-sitter#2596] [Q&A] [answered]
I have written a couple of Tree-sitter parsers, and recently I learned that Github.com uses Tree-sitter for syntax highlighting.

Is there an overview anywhere of which Tree-sitter parsers Github.com uses? It would be fun to brag about :slightly_smiling_face:

**Comment by amaanq** [comment, marked as answer]
https://github.com/github-linguist/linguist/tree/master/vendor the snail means tree-sitter

**Reply by Hubro** [reply]
Aha, very cool! Thanks :slightly_smiling_face:

**Reply by alexpovel** [reply]
> the snail means tree-sitter

Rude, ha!

**Reply by maxbrunsfeld** [reply]
yeah, seriously rude
